File: C/sequencia_impares.c
#include <stdio.h>
#include <locale.h>

int x;

int main() {
    setlocale(LC_ALL, "Portuguese");

    printf("Digite o valor de X: ");
    scanf("%d", &x);

    for (int i = 0; i <= x; i++) {
        if (i % 2 != 0) {
            printf("%d \n", i);
        }
    }

}

File: Java/sequencia_impares.java
import java.util.Locale;
import java.util.Scanner;

public class sequencia_impares {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        Locale.setDefault(Locale.US);

        System.out.print("Digite o valor de x: ");
        int x = sc.nextInt();

        for (int i = 0; i < x; i++) {
            if (i % 2 != 0) {
                System.out.printf("%d %n", i);
            }
        }

        sc.close();

    }
}
